### PR TITLE
concepts-cluster-administration-networking-pr-2017-09-25

### PR DESCRIPTION
**Pull request by JCai-Code**
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to  
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

### CODE REVIEW

File: docs/concepts/cluster-administration/networking.md [hunk 1]
```diff
@@ -18,8 +31,17 @@ default.  There are 4 distinct networking problems to solve:
 
 * TOC
 {:toc}
+-->
 
+Kubernetes对网络的处理方式与docker有些不同，有4个不同的网络问题需要解决：
```
Comment (markthink) [inline docs/concepts/cluster-administration/networking.md]: 中英之间用空格分隔

Comment (JCai-Code) [reply docs/concepts/cluster-administration/networking.md]: 是文章所有的中英之间都要加空格吗？

File: docs/concepts/cluster-administration/networking.md [hunk 2]
```diff
@@ -182,55 +311,123 @@ perfect the process, please let us know.
...
 ### OVN (Open Virtual Networking)
 
 OVN is an opensource network virtualization solution developed by the
 Open vSwitch community.  It lets one create logical switches, logical routers,
 stateful ACLs, load-balancers etc to build different virtual networking
 topologies.  The project has a specific Kubernetes plugin and documentation
 at [ovn-kubernetes](https://github.com/openvswitch/ovn-kubernetes).
+-->
+
+### OVN(开发的虚拟网络)
```
Comment (zjj2wry) [inline docs/concepts/cluster-administration/networking.md]: 开发的

**Review by zjj2wry — APPROVED** [review]
generate lgtm

**Pull request merged**